nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB/metadata.yaml

```diff
@@ -59,24 +59,24 @@ implemented_observables:
       - $W^-$
     plot_x: k1
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - abs_eta #nnpdf website says eta/y
```

Comment:
As before, the docs don´t really help here. Look at the variables in `process_options` and select the one that is closer to what you need.

In your case, indeed, I would say it is abs_eta since that's the actual kinematics `https://github.com/NNPDF/nnpdf/blob/541ab4d68692b0e570254aa071ddb7d99443889c/validphys2/src/validphys/process_options.py#L32`